fendermint/vm/topdown/src/voting.rs

```diff
@@ -418,6 +460,14 @@ pub async fn publish_vote_loop<V, F>(
                     if let Err(e) = client.publish_vote(vote) {
                         tracing::error!(error = e.to_string(), "failed to publish vote");
                     }
+
+                    let block_hash_hex = hex::encode(&next_hash);
+
+                    emit(ParentFinalityPeerVoteSent {
```

Comment:
why not handle hex conversion in the method? It says a lot of `hex::encode` and one does not really have to import hex because of tracing

Reply:
Good point!